common/model-views.cpp

```diff
@@ -1001,7 +1001,10 @@ namespace rs2
         yuy2rgb(std::make_shared<rs2::gl::yuy_decoder>()),
         depth_decoder(std::make_shared<rs2::depth_huffman_decoder>()),
         viewer(viewer),
-        detected_objects(device_detected_objects)
+        detected_objects(device_detected_objects),
+        SHORT_RANGE_MIN_DISTANCE(0.05f),
+        SHORT_RANGE_MAX_DISTANCE(1.5f)
```

Comment:
Check if it is possible to increase to 2 up to 3 meter and check how it applies to Colorizer and Threshold filters